js/components/portal-dashboard/feedback/activity-level-feedback-student-rows.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { Map } from "immutable";
+import { getFormattedStudentName } from "../../../util/student-utils";
+import AwaitingFeedbackActivityBadgeIcon from "../../../../img/svg-icons/awaiting-feedback-activity-badge-icon.svg";
+import GivenFeedbackActivityBadgeIcon from "../../../../img/svg-icons/given-feedback-activity-badge-icon.svg";
+import UpdateFeedbackQuestionBadgeIcon from "../../../../img/svg-icons/update-feedback-question-badge-icon.svg";
```

Comment:
`UpdateFeedbackQuestionBadgeIcon` is only used in the var below that doesn't end up being used.  If this is for future use, let's comment out for now to avoid the warnings.